src/SerialLoops/SectionList.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Eto.Forms;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using Eto.Drawing;
+
+/// Taken from https://github.com/picoe/Eto/blob/ac4610775b70538b96995bdb0c7f0fbcc5ae1b66/test/Eto.Test/SectionList.cs
+namespace SerialLoops
+{
+    public interface ISection
+    {
+        string Text { get; }
+    }
+
+    /// <summary>
+    /// Sections can nest. Each section item can also host
+    /// a control that is displayed in the details view when 
+    /// the section is selected.
+    /// 
+    /// Sections do not have any particular visual representation,
+    /// and can be wrapped within a tree item (SectionTreeItem) or
+    /// any other visual representation.
+    /// </summary>
+    public class Section : List<Section>, ISection
+    {
+        public string Text { get; set; }
+
+        public Section()
+        {
+        }
+
+        public Section(string text, IEnumerable<Section> sections)
+            : base(sections.OrderBy(r => r.Text, StringComparer.CurrentCultureIgnoreCase).ToArray())
+        {
+            this.Text = text;
+        }
+    }
+
+    /// <summary>
+    /// A tree item representation of a section.
+    /// </summary>
+    public class SectionTreeItem : List<SectionTreeItem>, ITreeGridItem<SectionTreeItem>
+    {
+        public Section Section { get; private set; }
+        public string Text { get { return Section.Text; } }
+        public bool Expanded { get; set; }
+        public bool Expandable { get { return Count > 0; } }
+        public ITreeGridItem Parent { get; set; }
+
+        public SectionTreeItem(Section section)
+        {
+            this.Section = section;
+            this.Expanded = true;
```

Comment:
```suggestion
            this.Expanded = false;
```

This will make it easier to browse the tree when a project is first opened.